packages/itwinui-css/src/table/base.scss

```diff
@@ -35,7 +34,7 @@
   scrollbar-color: transparent transparent; // Reserve space but never show the scrollbar
```

Comment:
I think this can be removed now? As well as the `table-scrollbar` mixin

Reply:
Removed `scrollbar-color` (https://github.com/iTwin/iTwinUI/pull/1891/files/730397d6ee1fac7d0275263ad1f15c9e3c6205fa..a52efb2654dbf217cb95af7e60e6a13bcd50f239) and the `table-scrollbar` mixin (https://github.com/iTwin/iTwinUI/pull/1891/commits/08c939449e7fa266f35780abf40b7e86c849a175).

Would be great if you or @AdamMeza-Bentley can confirm that everything still works on Windows + Chromium. (Related PR: https://github.com/iTwin/iTwinUI/pull/1816)

Reply:
Looks good

![image](https://github.com/iTwin/iTwinUI/assets/9084735/45fe4b76-1c0d-4009-a6da-544b27ecf4eb)

![image](https://github.com/iTwin/iTwinUI/assets/9084735/8ae3206e-b7b8-4973-ae7f-e38b4510ade6)

![image](https://github.com/iTwin/iTwinUI/assets/9084735/049337b8-9e58-42b3-ada0-a4aecf69bb53)